js/colors/utils.ts

```diff
@@ -0,0 +1,22 @@
+import { RGBColor, rgb } from "d3-color";
+
+// Color object used internally by 3D rendering. RGB values are within [0, 1] range.
+export type RGBAFloat = { r: number; g: number; b: number; a: number; };
+
+const toFloatRGB = 1 / 255;
+export function d3RGBToRGBAFloat(d3Rgb: RGBColor): RGBAFloat {
+  return { r: d3Rgb.r * toFloatRGB, g: d3Rgb.g * toFloatRGB, b: d3Rgb.b * toFloatRGB, a: d3Rgb.opacity };
+}
+
+export function cssColToRGBAFloat(cssColorSpecifier: string): RGBAFloat {
```

Comment:
```suggestion
export function cssColorToRGBAFloat(cssColorSpecifier: string): RGBAFloat {
```
You can afford the extra two characters. 😉

Reply:
Done ;)